src/services/mac-files.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Observable } from 'rxjs';
+import { spawn } from 'child_process';
+import { StreamService } from './stream.service';
+import { map } from 'rxjs/operators';
+import { UnixFilesService } from './unix-files.service';
+
+export class MacFilesService extends UnixFilesService {
+  constructor(streamService: StreamService) {
+    super(streamService);
+  }
```

Comment:
This constructor can be removed, it's inherited from `UnixFilesService`

```suggestion
```

Reply:
Done in 7494bac